input/pagecontent/reference.md

```diff
@@ -1842,6 +1836,42 @@ Any FSH syntax errors that arise as a result of the value substitution are handl
   // more rules
   ```
 
+##### Inserting Rule Sets at Elements
+{%include tu-div.html%}
+To insert a rule set at an element, specify the path to that element before the insert rule:
+
+<pre><code>* &lt;element&gt; insert {RuleSet name}(value1<i>, value2, value3...</i>)
+</code></pre>
```

Comment:
I think the whole parenthetical should be italics since it is entirely optional.
```suggestion
<pre><code>* &lt;element&gt; insert {RuleSet name}<i>(value1, value2, value3...</i>)
</code></pre>
```

Reply:
Agree, but the close tag should be after the `)`.  Change made.